decidim-core/app/views/decidim/shared/_filters.html.erb

```diff
@@ -1,32 +1,37 @@
-<%# REDESIGN_PENDING: Same logic as Meetings %>
+<% filter_sections = [] unless local_assigns.has_key?(:filter_sections) %>
+<% search_label = t("decidim.searches.filters.search") unless local_assigns.has_key?(:search_label) %>
 
-<%= redesigned_filter_form_for filter, url_for, class: "self-stretch", data: { filters: "", component: "accordion" } do |form| %>
+<% if filter_sections.present? || local_assigns.has_key?(:search_variable) %>
+  <%= redesigned_filter_form_for filter, url_for, class: "new_filter self-stretch", data: { filters: "", component: "accordion" } do |form| %>
 
-  <%
-    # REDESIGN_PENDIGN: fetch the process filters
-    filters = [];
-  %>
+    <button id="dropdown-trigger" data-component="dropdown" data-target="dropdown-menu-filters">
+      <%= icon "arrow-down-s-line" %>
+      <%= icon "arrow-up-s-line" %>
+      <span>
+        <%= local_assigns.has_key?(:search_variable) ? t("filter_and_search", scope: "decidim.searches.filters_small_view") : t("filter", scope: "decidim.searches.filters_small_view") %>
+      </span>
+    </button>
 
-  <% filters.each do |f| %>
-    <div class="filter-container">
-      <button data-controls="panel-dropdown-menu-<%= f[0] %>" data-open="true">
-        <%= icon "arrow-down-s-line" %>
-        <%= icon "arrow-up-s-line" %>
-        <span><%= t(f[0], scope: f[4]) %></span>
-      </button>
-      <div id="panel-dropdown-menu-<%= f[0] %>">
-        <%= form.send(
-            f[1] == "radio" ? "collection_radio_buttons" : "collection_check_boxes",
-            f[2],
-            f[3],
-            :first,
-            :last
-        ) do |builder|
-              builder.label {
-                builder.send(f[1] == "radio" ? "radio_button" : "check_box", class: "reset-defaults" , hidden: true ) + content_tag(:div, builder.text, class: "filter" ) }
-          end %>
-      </div>
+    <div id="dropdown-menu-filters">
+      <% if local_assigns.has_key?(:search_variable) %>
+        <div class="filter-search filter-container">
+          <%= form.search_field search_variable,
+            label: false,
+            placeholder: search_label,
+            title: search_label,
+            "aria-label": search_label %>
+          <button type="submit" aria-controls="meetings" aria-label="<%= search_label %>">
```

Comment:
Is it ok to have the `aria-controls="meetings"` attribute ?

Reply:
Nope, you're right. I'll take it out

Reply:
Done